code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -426,6 +427,14 @@
 		spawn(rand(0, 15))
 			stat |= NOPOWER
 
+/obj/machinery/chem_master/emag_act(mob/user)
+	if(!safety_override)
+		to_chat(user, "<span class='notice'>You disable the safeties on the [src].</span>")
+		safety_override = 1
+	else
+		to_chat(user, "<span class='notice'>You re-enable the safeties on the [src].</span>")

```

Comment:
I thought the standard was for emags to be permanent. That way, you have a risk of detection.